src/equations/compressible_euler_quasi_1d.jl

```diff
@@ -0,0 +1,379 @@
+    a_rho, a_rho_v1, a_e, a = u
+    rho, v1, p, a = cons2prim(u, equations)
+    e = a_e / a
+
+    # Ignore orientation since it is always "1" in 1D
+    f1 = a_rho_v1
+    f2 = a_rho_v1 * v1
+    f3 = a * v1 * (e + p)
+
+    return SVector(f1, f2, f3, zero(eltype(u)))
+end
+
+"""
+@inline function flux_nonconservative_chan_etal(u_ll, u_rr, orientation::Integer,
+                                                equations::CompressibleEulerEquationsQuasi1D)
+
+Non-symmetric two-point volume flux discretizing the nonconservative (source) term
+that contains the gradient of the pressure  [`CompressibleEulerEquationsQuasi1D`](@ref) 
+and the nozzle width.
+
+Further details are available in the paper:
+- Jesse Chan, Khemraj Shukla, Xinhui Wu, Ruofeng Liu, Prani Nalluri (2023)
+    High order entropy stable schemes for the quasi-one-dimensional
+    shallow water and compressible Euler equations
+    [DOI: 10.48550/arXiv.2307.12089](https://doi.org/10.48550/arXiv.2307.12089)    
+"""
+@inline function flux_nonconservative_chan_etal(u_ll, u_rr, orientation::Integer,
+                                                equations::CompressibleEulerEquationsQuasi1D)
+    #Variables
+    _, _, p_ll, a_ll = cons2prim(u_ll, equations)
+    _, _, p_rr, _ = cons2prim(u_rr, equations)
+
+    # For flux differencing using non-conservative terms, we return the 
+    # non-conservative flux scaled by 2. This cancels with a factor of 0.5 
+    # in the arithmetic average of {p}.
+    p_avg = p_ll + p_rr
+
+    z = zero(eltype(u_ll))
+
+    return SVector(z, a_ll * p_avg, z, z)
+end
+
+"""
+@inline function flux_chan_etal(u_ll, u_rr, orientation::Integer,
+                                           equations::CompressibleEulerEquationsQuasi1D)
+
+Conservative (symmetric) part of the entropy conservative flux for quasi 1D compressible Euler equations split form.
+This flux is a generalization of [`flux_ranocha`](@ref) for [`CompressibleEulerEquations1D`](@ref).
+Further details are available in the paper:
+- Jesse Chan, Khemraj Shukla, Xinhui Wu, Ruofeng Liu, Prani Nalluri (2023) 
+  High order entropy stable schemes for the quasi-one-dimensional
+  shallow water and compressible Euler equations
+  [DOI: 10.48550/arXiv.2307.12089](https://doi.org/10.48550/arXiv.2307.12089)     
+"""
+@inline function flux_chan_etal(u_ll, u_rr, orientation::Integer,
+                                equations::CompressibleEulerEquationsQuasi1D)
+    # Unpack left and right state
+    rho_ll, v1_ll, p_ll, a_ll = cons2prim(u_ll, equations)
+    rho_rr, v1_rr, p_rr, a_rr = cons2prim(u_rr, equations)
+
+    # Compute the necessary mean values
+    rho_mean = Trixi.ln_mean(rho_ll, rho_rr)
+    # Algebraically equivalent to `inv_ln_mean(rho_ll / p_ll, rho_rr / p_rr)`
+    # in exact arithmetic since
+    #     log((ϱₗ/pₗ) / (ϱᵣ/pᵣ)) / (ϱₗ/pₗ - ϱᵣ/pᵣ)
+    #   = pₗ pᵣ log((ϱₗ pᵣ) / (ϱᵣ pₗ)) / (ϱₗ pᵣ - ϱᵣ pₗ)
+    inv_rho_p_mean = p_ll * p_rr * Trixi.inv_ln_mean(rho_ll * p_rr, rho_rr * p_ll)
```

Comment:
I suppose `Trixi.` is also not necessary here.

```suggestion
    inv_rho_p_mean = p_ll * p_rr * inv_ln_mean(rho_ll * p_rr, rho_rr * p_ll)
```